rabbitmq/tests/test_openmetrics.py

```diff
@@ -523,3 +523,84 @@ def test_detailed_endpoint_queue_coarse_metrics(aggregator, dd_run_check, mock_h
             "tags": ["endpoint:localhost:15692/metrics/detailed?family=queue_coarse_metrics"] + m.get('tags', []),
         }
         aggregator.assert_metric(**kwargs)
+
+
+def test_per_object(aggregator, dd_run_check, mock_http_response):
+    """We scrape the /metrics/per-object endpoint."""
+    mock_http_response(file_path=OPENMETRICS_RESPONSE_FIXTURES / "per-object.txt")
+    check = RabbitMQ(
+        "rabbitmq",
+        {},
+        [
+            {
+                'prometheus_plugin': {
+                    'url': "localhost:15692",
+                    'unaggregated_endpoint': 'per-object',
+                },
+                "metrics": [".+"],
+            }
+        ],
+    )
+    dd_run_check(check)
+
+    expected_metrics = (
```

Comment:
Same comment as one in an earlier PR, to use the built-in aggregator assert methods to simplify the test: https://github.com/DataDog/integrations-core/pull/13684#discussion_r1069772419

Reply:
see #13737